ubi_manifest/worker/tasks/content_audit.py

```diff
@@ -113,13 +113,13 @@ def content_audit_task() -> None:
                         out_repo.ubi_config_version,
                     )
                     output_blacklist.extend(parse_blacklist_config(config))
-                    whitelist, debuginfo_whitelist = filter_whitelist(
-                        config, output_blacklist
-                    )
-                    output_whitelist |= whitelist | debuginfo_whitelist
-                    output_whitelist |= {
-                        f"{md.name}:{md.stream}" for md in config.modules.whitelist
-                    }
+                    pkg_whitelist, _ = filter_whitelist(config, output_blacklist)
```

Comment:
So are we now stopping to check the debug repos content? I don't think we wanted to skip this as well.

If we agreed about that somewhere, please point me there. I am getting totally lost with this audit thing.